# Custom message for bike speed data
std_msgs/Header header
float32 speed           # Calculated speed in m/s
float32 distance        # Calculated distance in meters
